src/gui/updownratiodlg.h

```diff
@@ -24,29 +24,26 @@
  * modify file(s), you may extend this exception to your version of the file(s),
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
- *
- * Contact : chris@qbittorrent.org
  */
 
 #ifndef UPDOWNRATIODLG_H
 #define UPDOWNRATIODLG_H
 
 #include <QDialog>
 
-QT_BEGIN_NAMESPACE
-namespace Ui {
+namespace Ui
+{
     class UpDownRatioDlg;
 }
-QT_END_NAMESPACE
 
 class UpDownRatioDlg : public QDialog
 {
     Q_OBJECT
 
 public:
     explicit UpDownRatioDlg(bool useDefault, qreal initialValue, qreal maxValue,
```

Comment:
`explicit` is redundant in this declaration. Please delete it, since you're here.